Assets/Scripts/Networking/CustomNetworkManager.cs

```diff
@@ -59,26 +56,20 @@ void OnLoaderFinished()
                 return;
             }
 
+
             // make client ready
-            if (ClientScene.ready)
+            if (NetworkClient.ready)
                 Debug.LogErrorFormat("Client was made ready before loader finished");
             else
-                ClientScene.Ready(NetworkClient.connection);
+            {
+                NetworkClient.Ready();
+            }
 
             // add player if specified
-            if (autoCreatePlayer && ClientScene.localPlayer == null)
+            if (autoCreatePlayer && NetworkClient.localPlayer == null)
             {
-                ClientScene.AddPlayer();
+                NetworkClient.AddPlayer();
             }
         }
-
-        public override void ConfigureServerFrameRate()
```

Comment:
from Mirror docs: ConfigureServerFrameRate was renamed to ConfigureHeadlessFrameRate.

so we need to override it, or otherwise it will assign frame rate